corehq/apps/export/views/list.py

```diff
@@ -333,6 +330,12 @@ def _fmt_emailed_export_fileData(self, fileId, size, last_updated,
             'downloadUrl': download_url,
         }
 
+    def _get_owner_username(export):
+        user = None
+        if export.owner_id:
+            user = CouchUser.get_by_user_id(export.owner_id)
+        return user.username if user else UNKNOWN_EXPORT_OWNER
```

Comment:
It feels like there should be a "right" way to structure this function (re: discussion on Slack), but it also seems fine as is.

Reply:
It made it to its final form (hopefully) f17a4380e33d1e3e3329f3e4a24f0a065de72190